tools/oio-election-reset.py

```diff
@@ -30,6 +30,25 @@
         "sqlx":  (2,2),
 }
 
+class ZkHandle(object):
+    def __init__(self, zh):
+        self._zh = zh
+
+    def get(self):
+        return self._zh
+
+    def close(self):
+        zookeeper.close(self._zh)
+        self._zh = None
+
+
+def get_connected_handles(cnxstr):
+    zookeeper.set_debug_level(zookeeper.LOG_LEVEL_INFO)
+    for shard in cnxstr.split(";"):
+        zh = zookeeper.init(shard)
+    yield ZkHandle(zh)
```

Comment:
Looks like this statement should be in the loop.

Reply:
In facts, the whole has not been deduplicated at all.